packages/material-ui/test/mui-5/Form.test.tsx

```diff
@@ -232,4 +232,11 @@ describe("single fields", () => {
       .toJSON();
     expect(tree).toMatchSnapshot();
   });
+  test("using custom tagName", () => {
```

Comment:
We should add this test to all themes if possible

Reply:
Will do!